static/themes/default/theme.css

```diff
@@ -379,14 +390,14 @@
     border-bottom: 1px solid #c0c0c0;
 }
 
-.kiwi-messagelist-message.kiwi-messagelist-message-connection .kiwi-messagelist-body,
-.kiwi-messagelist-message.kiwi-messagelist-message-disconnected .kiwi-messagelist-body {
+.kiwi-messagelist-message.kiwi-messagelist-message-connection .kiwi-messagelist-body {
```

Comment:
This selector can be shortened down to `.kiwi-messagelist-message-connection .kiwi-messagelist-body`